x/pairing/keeper/unresponsive_provider.go

```diff
@@ -11,10 +11,11 @@ import (
 )
 
 // Function that returns a map that links between a provider that should be punished and its providerCuCounterForUnreponsiveness
-func (k Keeper) UnstakeUnresponsiveProviders(ctx sdk.Context, epochsNumToCheckCUForUnresponsiveProvider uint64, epochsNumToCheckCUForComplainers uint64) error {
+func (k Keeper) UnstakeUnresponsiveProviders(ctx sdk.Context, epochsNumToCheckCUForUnresponsiveProvider uint64, epochsNumToCheckCUForComplainers uint64) {
 	// check the epochsNum consts
 	if epochsNumToCheckCUForComplainers <= 0 || epochsNumToCheckCUForUnresponsiveProvider <= 0 {
-		return utils.LavaFormatError("epochsNumToCheckCUForUnresponsiveProvider or epochsNumToCheckCUForComplainers are smaller or equal than zero", fmt.Errorf("invalid unresponsive provider consts"),
+		utils.LavaFormatError("epoch to check CU for unresponsive provider or for complainer is zero",
```

Comment:
if an error was found lets return here since the logic afterwards will be broken

Reply:
this is called from EpochStart(), in turn called from BeginBlock() - there is nothing to do anything with this error except log it (which is what the original code does). In other words - if there is concern about breaking things in the future then we need to panic here.